sidebar.md

```diff
@@ -52,24 +52,18 @@ We actively recruit people from diverse time zones and backgrounds, who have goo
 ##Filter Posts
 [MOD POST](https://www.reddit.com/r/AsianBeauty/search?q=+flair:mod-post&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [REVIEW](https://www.reddit.com/r/AsianBeauty/search?q=+flair:review&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[PSA](https://www.reddit.com/r/AsianBeauty/search?q=+flair:psa&restrict_sr=on&sort=new&t=all&feature=legacy_search)

```

Comment:
Are we deprecating PSA entirely or will the modteam still be using it for selected mod posts? (I vote deprecate entirely)